packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/AppConfiguration.kt

```diff
@@ -106,6 +113,47 @@ public interface AppConfiguration {
                 this.userLoggers = customLoggers
             }
 
+        /**
+         * Configures the root folder containing all files and realms used when synchronizing data
+         * between the device and MongoDB Realm.
+         *
+         * The default root directory is platform-dependent:
+         * ```
+         * // For Android the default directory is obtained using
+         * val dir = ${Context.getFilesDir()/mongodb-realm}
```

Comment:
Nitpick...but this isn't actually valid Kotlin code...these should be encapsulated in `""` for that.

Reply:
🙈